home_work_3/seminar_task_7.py

```diff
@@ -0,0 +1,26 @@
+# Задание №7 семинара 3. Count.
+
+my_str = input('Введите строку: ')
+print(my_str)
+my_str = my_str.lower()
+my_str = my_str.replace(' ', '')
+my_dict1 = {}
+my_dict2 = {}
+
+for item in my_str:
+    counter = 0
+    if item == ' ':
+        continue
+    for element in my_str:
+        if item == element:
+            counter += 1
+    my_dict1[item] = counter
```

Comment:
Попробуйте использовать метод `get` решение будет значительно проще.
```suggestion
for item in my_str.split():
    my_dict1[item] = my_dict1.get(item, 0) + 1
```